db/schema.rb

```diff
@@ -204,6 +204,8 @@
     t.string "temp_reuse_address_for_site_location"
     t.boolean "temp_use_registered_company_details"
     t.datetime "companies_house_updated_at"
+    t.boolean "temp_reuse_applicant_name"
```

Comment:
Please check why this change crept in. Possibly related to ongoing issues around WEX migrations.

Reply:
The `temp_reuse_applicant_name` change was merged into the engine on July 11th. The back office schema hasn't been updated with this change since the merge. I ran a db:migrate due to issues with my db so I think that's why this PR has the updated schema.

Reply:
I think the front office needs the schema updating as well